kernel/packages/decentraland-loader/lifecycle/controllers/parcel.ts

```diff
@@ -35,24 +37,30 @@ export class ParcelLifeCycleController extends EventEmitter {
 
     this.currentPosition = position
 
+    return this.updateLoadedParcels(this.config.secureRadius)
+  }
+
+  updateLoadedParcels(secureRadius: number = 0): ParcelSightSeeingReport | undefined {
+    if (this.currentPosition === undefined) return undefined
+
     this.isTargetPlaced = true
-    const sightedParcels = parcelsInScope(this.config.lineOfSightRadius, position)
+    const sightedParcels = parcelsInScope(this.config.lineOfSightRadius, this.currentPosition)
     const sightedParcelsSet = new Set<string>()
 
-    const newlySightedParcels = sightedParcels.filter(parcel => {
+    const newlySightedParcels = sightedParcels.filter((parcel) => {
       sightedParcelsSet.add(parcel)
       return this.parcelSighted(parcel)
     })
 
-    const secureParcels = new Set(parcelsInScope(this.config.lineOfSightRadius + this.config.secureRadius, position))
+    const secureParcels = new Set(parcelsInScope(this.config.lineOfSightRadius + secureRadius, this.currentPosition))
```

Comment:
We might want to review the `secureRadius` parameter later. Right now the secure/LOS ratio is 1:1, but with this change the LOS can be 10 and the `secureRadius` would remain at 4.

Reply:
Yes, I thought of the same, I think ideally we should have a secureRadius relative to the LOS, like 1/2 of the LOS or less.

Do you want me to try that? I could use a floored `this.config.lineOfSightRadius / 3` and see how that works (It would change the regular secure radius from 4 to 3, and keep it relative to the LOS from now on)

Reply:
Done